docs/ref/contrib/postgres/fields.txt

```diff
@@ -270,6 +272,39 @@ A more useful index is a ``GIN`` index, which you should create using a
     .. _citext: https://www.postgresql.org/docs/current/static/citext.html
     .. _the performance considerations: https://www.postgresql.org/docs/current/static/citext.html#AEN169274
 
+``CICharField``
+===============
+
+.. class:: CICharField(**options)
```

Comment:
What do you think about including these under the CIText heading? I think it's not adding much value to repeat such similar text 3 times.